conf/main.yaml

```diff
@@ -0,0 +1,13 @@
+# List of configuration files loaded in usmqe test framework.
+# It is recomended to keep defaults.yaml configuration file and attach more
+# configuration files.
+#
+# Each item in list overwrites previous configuration file.
+#
+configuration_files:
+  - conf/defaults.yaml
+  - conf/example-usm1.yaml
```

Comment:
Since we suggest to prepare the config file via `cp conf/example_usm.yaml conf/usm.yaml`, I would include the `usm.yaml` directly here, instead of the `example-usm1.yaml`. Then it will require to change only the `usm.yaml` config file (in typical/default/ci environment).
Also there is slightly mishmash between the names of the example config file:
* `conf/example-usm1.yaml`
* `conf/example_usm.yaml`
and also:
* `conf/example-usm1.hosts`
* `conf/example_usm.hosts`

Reply:
Ok, renamed:
```
conf/example-usm1.yaml -> conf/usm.yaml
conf/example-usm1.hosts -> conf/usm.hosts
```

Reply:
I think, it is worth to keep the `example` word in the names of `usm.yaml` and `usm.hosts` files.
Those files are real examples, it is not supposed to use them directly.
So I'm proposing to:
* rename the example configuration files `usm.yaml` and `usm.hosts` to `example_usm.yaml` and `example_usm.hosts`,
* keep the `usm.yaml` and `usm.hosts` in the list of included files in `main.yaml`.